apps/core/models.py

```diff
@@ -186,6 +186,12 @@ class Meta:
     def __str__(self):
         return str(self.reference)
```

Comment:
I would prefer a composite str like (see comments below for more details about `UUIDModelMixin`):

```python
class OmicsUnit(UUIDModelMixin, models.Model):

    def __str__(self):
        return '{} ({}/{})'.format(
            self.get_short_uuid(),
            self.type,
            self.strain,
        )
```